### ISSUE TITLE
Yomu Mangás

### ISSUE THREAD

**Issue by SrCrono** [KotatsuApp/kotatsu-parsers#332]
### Source name

Yomu Mangás

### Source link

https://yomumangas.com/

### Language

Brazilian Portuguese

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have checked that the source does not already exist on the app.
- [X] I have checked that the source does not already exist by searching the [GitHub repository](https://github.com/KotatsuApp/kotatsu-parsers) and verified it does not appear in the code base.

**Comment by davvarrr** [comment]
Error 503 source are woks ?

**Comment by deividgabrielpeira** [comment]
> Error 503 source are woks ?

Yes, the site still works, possibly the ip where you live has been blocked.
